grpc/testing/messages.proto

```diff
@@ -207,20 +206,29 @@ message LoadBalancerStatsRequest {
   int32 num_rpcs = 1;
   // If num_rpcs have not completed within timeout_sec, return partial results.
   int32 timeout_sec = 2;
-  // response header+trailer we want the values of
+  // Response header + trailer metadata entries we want the values of.
+  // Matching of the keys is case-insensitive as per rfc7540#section-8.1.2
+  // * (asterisk) is a special value that will return all metadata entries
   repeated string metadata_keys = 3;
 }
 
 message LoadBalancerStatsResponse {
+  enum MetadataType {
+    Initial = 0;
+    Trailing = 1;
+  }
   message MetadataEntry {
+    // Key, exactly as received from the server. Case may be different from what
+    // was requested in the LoadBalancerStatsRequest)
     string key = 1;
+    // Value, exactly as received from the server.
     string value = 2;
+    // Metadata type is either Initial or Trailing
+    MetadataType type = 3;
   }
   message RpcMetadata {
     // metadata values for each rpc for the keys specified in
     // LoadBalancerStatsRequest.metadata_keys.
-    // metadata keys and values are returned exactly as was received
```

Comment:
Why is this comment removed? Why is it no longer true (that "metadata keys and values are returned exactly as was received")?

Reply:
I moved the comments above, to metadata entry. I also elaborated there that the key has same case as server supplied, and not the value from the original request. I decided not to duplicate the comment so there's no unsyncing in the future.